src/mmw/js/src/analyze/models.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var $ = require('jquery'),
-    _ = require('lodash'),
+    lodash = require('lodash'),
```

Comment:
Having lodash aliased to `_` caused an odd scoping problem that I think was due to a naming conflict with underscore (also aliased to `_`), which resulted in anything in the lodash namespace not being in scope.